kotlin/samples/hello-terminal/app/README.md

```diff
@@ -0,0 +1,7 @@
+# hello-terminal / app
```

Comment:
More depth explaining what's going on here, in particular where lanterna ends and the sample code begins.

Reply:
Rather than two README files in `app` and the library, probably easier to find a single one up in `hello-terminal`.

Reply:
Fleshed out the docs a little, lemme know what you think.